packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts

```diff
@@ -1,4 +1,4 @@
 export enum NftStandard {
     Irc27 = 'IRC27',
-    Erc721 = 'ERC721',
+    Erc721 = 'ERC-721',
```

Comment:
@Tuditi why are we making this change?

Reply:
To follow the specification.
TIP: 
![image](https://github.com/bloomwalletio/bloom/assets/45079109/35900427-2a36-40eb-bb49-7fde2bc64c90)
EIP:
![image](https://github.com/bloomwalletio/bloom/assets/45079109/fc02afa1-f31c-4ddb-bd0a-5e65c456ba34)